renderlib/FileReaderTIFF.cpp

```diff
@@ -372,7 +411,7 @@ readTiffPlane(TIFF* tiff, int planeIndex, const VolumeDimensions& dims, uint8_t*
       return false;
     }
     // copy buf into data.
-    size_t numBytesCopied = copyAndConvert(dataPtr, reinterpret_cast<uint8_t*>(buf), numBytesRead, dims.bitsPerPixel);
+    size_t numBytesCopied = copyDirect(dataPtr, reinterpret_cast<uint8_t*>(buf), numBytesRead, dims.bitsPerPixel);
```

Comment:
You are using reinterpret_cast a lot. I looked it up and am a little concerned that the consensus is it makes your code non-portable. Can you comment on this?

Reply:
hmm, thanks - I think you're right to flag this.  Going from void* to uint8_t* is totally reasonable for static_cast.  I will make that change.